adlfs/spec.py

```diff
@@ -892,6 +901,7 @@ async def _ls(
         delimiter: str = "/",
         return_glob: bool = False,
         version_id: Optional[str] = None,
+        versions: bool = False,
```

Comment:
Turns out `_ls` doesn't support `detail` (even though it states that it does in the docstring). I'll create a pre-requisite PR for that right now (also `_ls` is very crowded and hard to read).